### PR TITLE
feat: chatbot

### LINKED ISSUE #5: [FEATURE] - Chat Bot
Implementing frontend for [IEEE-FAQ-Chatbot](https://github.com/IEEE-VIT/IEEE-FAQ-Chatbot)

### PR DESCRIPTION
**Pull request by aneeshpanda**
Implemented the chatbot.

Closes #5

Screenshot 2021-08-20 144815

### CODE REVIEW

**Review by r-ush — APPROVED** [review]
LGTM! Waiting for review from everyone

**Comment by r-ush** [comment]
Going live today, LMK if you want to make any changes/improvements before its live @aneeshpanda

**Comment by aneeshpanda** [comment]
> Going live today, LMK if you want to make any changes/improvements before its live @aneeshpanda

Nopes, nothing from my side. LGTM.

**Pull request merged**
